urllib3/response.py

```diff
@@ -269,7 +270,18 @@ def read(self, amt=None, decode_content=None, cache_content=False):
             except HTTPException as e:
                 # This includes IncompleteRead.
                 raise ProtocolError('Connection broken: %r' % e, e)
+        except Exception:
```

Comment:
**Do not use `except:`**.

Not all exceptions derive from `Exception`, and that's why we use `except Exception`. Python implements some very important things as exceptions, like Ctrl-C-ing a program (`KeyboardInterrupt`) and calling `sys.exit()` (`SystemExit`). Those *are* exceptions and so will be caught by `except:`, but they do *not* inherit from `Exception` (they inherit from `BaseException`). You want to let them bubble up, so `except Exception:` is absolutely the idiomatic thing to do.